integrator/integrator.py

```diff
@@ -0,0 +1,98 @@
+from sys import argv
+from subprocess import Popen
+import os
+from integratorlib import *
+
+
+# Pack size for integrator
+kmax=1000
+
+# Checking input format
+lenarg=len(argv)
+if lenarg < 2 : quit('Error! No arguments found. Please follow the format described in documentation')
+# If only one asteroid in input
+if lenarg == 2 :
+    astlist=[argv[1]]
+# If "-list" or "-range" or neither keys are specified
+if lenarg > 2 :
+    if argv[1]=='-list':
+        astlist=argv[2:]
+    elif argv[1]=='-range':
+        try:
+            astlist=[str(i) for i in range(int(argv[2]),int(argv[3])+1)]
+        except:
+            quit('Error! Wrong range arguments. Please follow the format described in documentation')
+    else:
+        astlist=argv[1:]
+
+
+# Open source file
+statinfo = os.stat('../asteroids_format.csv').st_size
+f=open('../asteroids_format.csv','r')
+# Length of a record in source file (must be constant)
+stlen=len(f.readline())
+if statinfo%stlen != 0:
+    quit('Error! Source file with orbital elements is not formatted, unable to launch searching.')
+# Number of records in source
+lrec=statinfo//stlen
+# Get orbital elements of a given list
+records=[]
+for i in astlist:
+    # And check if creating some .aei-s is not required
+    try:
+        probe=open('../aeibase/'+i+'.aei','r')
+        probe.close()
+        print('\t.aei file for asteroid ',i,' already exists.')
+    except:
+        records.append(find_asteroid(i,f,lrec,stlen))
+f.close()
+
+
+# Small.in header
+smallin=[
+    ')O+_06 Small-body initial data  (WARNING: Do not delete this line!!)\n',
+    ') Lines beginning with `)\' are ignored.\n',
+    ')---------------------------------------------------------------------\n',
+    ' style (Cartesian, Asteroidal, Cometary) = Asteroidal\n',
+    ')--------------------------------------------d or D is not matter--0d0 is possible too--\n',' \n']
+if len(smallin)<6:
```

Comment:
how is it possible? You define smallin array a line before. Why do you check the number of lines. It can't be less than 6.